client/vnctl/lib/vnctl/cli/base.rb

```diff
@@ -242,7 +242,7 @@ def self.define_mode_relation(mode_type, required_opts = [], &block)
           }
 
           yield self if block_given?
-          desc "del #{base_uuid.upcase}_UUID" , "Removes a(n) #{mode_type} #{base_uuid}."
+          desc "del #{base_uuid.upcase}_UUID --option=OPTION", "Removes a(n) #{mode_type} #{base_uuid}."
```

Comment:
That's one way of doing it but for consistency's sake, I think it's better to stay with the `[OPTIONS]` way of writing it. That's how it's currently written everywhere else in vnctl.